src/composables/accounts.ts

```diff
@@ -26,11 +27,33 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
     return accountsAddressList.value.includes(address);
   }
 
+  /**
+   * @param {string} address - the address to format.
+   * @returns {string} - returns the account's name if it exists,
+   * otherwise it returns the translated string "Account" + the account's index +1.
+  */
+  function getAccountNameByAddress(address: string): string {
+    const { idx, name } = accounts.value.find(
+      (acc) => acc.address === address,
+    ) || {};
```

Comment:
I suggest to create separate function for this, eg.: `getAccountByAddress`. I'm asking for this because I would use it in my current branch.